discovery.formatted.yaml

```diff
@@ -572,6 +572,19 @@ definitions:
         description: TBD
         type: boolean
     type: object
+  QuayRepoManifest:
+    description: Dockstore observed return objec
+    properties:
+      digest:
+        description: TBD
```

Comment:
I know I didn't do this before, but given that these are needed/used for this fix, might be a good time to fill in the descriptions